docker/Dockerfile

```
@@ -0,0 +1,74 @@
+RUN service postgresql start && service postgresql status \
+&& sudo --preserve-env=AIRFLOW__DATABASE__SQL_ALCHEMY_CONN --login -u airflow airflow db init
+
+# configure airflow
+USER airflow
+RUN sed -i "s|sqlite:////home/airflow/airflow/airflow.db|"$AIRFLOW__DATABASE__SQL_ALCHEMY_CONN"|" /home/airflow/airflow/airflow.cfg
+RUN sed -i "s|SequentialExecutor|LocalExecutor|" /home/airflow/airflow/airflow.cfg
+RUN sed -i "s|load_examples = True|load_examples = False|" /home/airflow/airflow/airflow.cfg
+
+# initialize airflow database
+USER root
+RUN service postgresql start && service postgresql status \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)